glue_jupyter/bqplot/common/tools.py

```diff
@@ -41,6 +41,36 @@ def __init__(self, viewer, **kwargs):
                                         'y': [self.viewer.scale_y]})
 
 
+@viewer_tool
+class BqplotPanZoomXMode(InteractCheckableTool):
+
+    icon = 'glue_move_x'
+    tool_id = 'bqplot:panzoom_x'
+    action_text = 'Pan and Zoom'
```

Comment:
Should this text be the same, or should it be mention X

Reply:
Good call, updated this and the Y tool